proto/src/determined/agent/v1/agent.proto

```diff
@@ -22,8 +22,6 @@ message Agent {
   map<string, Slot> slots = 3;
   // A map of container id to all containers assigned to this agent.
   map<string, determined.container.v1.Container> containers = 4;
-  // An optional label applied to the agent for scheduling restrictions.
```

Comment:
I'd recommend leaving the protos unchanged.  This seems like a pointless API breakage.

What could happen is a new cli would break when asking an old master about the agents on the cluster.  If you leave the field then no breakage happens.